pegasus/sites.v3/code.org/views/csp_information.md.erb

```diff
@@ -0,0 +1,25 @@
+<% facebook = {:u=>'https://youtu.be/DMr1iFYacGQ'} %>
+<% twitter = {:url=>'https://youtu.be/DMr1iFYacGQ', :related=>'codeorg', :text=>"Computer Science Principles. (Thanks #{get_random_donor_twitter} for supporting @codeorg)"} %>
+
+<div class="col-40" style="float:left; padding:10px;">
+
+  <%=view :display_video_thumbnail, id: "CSPrinciples", video_code: "DMr1iFYacGQ", play_button: 'center', facebook: facebook, twitter: twitter, letterbox: 'false' %>
+
+</div>
+
+[col-60]
+
+Designed for 9 - 12 grade students, CS Principles introduces students to the foundational concepts of computer science and challenges them to explore how computing and technology can impact the world. This year-long course can be taught as an introductory course and as an AP course - no prerequisites required for students or teachers new to computer science! CS Principles complements CS Discoveries with a deeper focus on concepts such as how the internet works and the societal impacts of computer science.
+
+<% facebook = {:u=>'https://youtu.be/DMr1iFYacGQ'} %>
+<% twitter = {:url=>'https://youtu.be/DMr1iFYacGQ', :related=>'codeorg', :text=>"Computer Science Principles. (Thanks #{get_random_donor_twitter} for supporting @codeorg)"} %>
```

Comment:
are these necessary?

Reply:
They do enable sharing options on the videos, so I think they're somewhat useful to keep.

Reply:
oh, I specifically mean: is it necessary to redeclare the variables here, after also declaring them at the top of the file?

Reply:
Oops, you're right.  Will fix.